src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsMoreActions/ContactDetailsMoreActions.test.tsx

```diff
@@ -54,15 +54,14 @@ describe('ContactDetailsMoreActions', () => {
         <TestRouter router={router}>
           <ThemeProvider theme={theme}>
             <GqlMockedProvider>
-              <ContactsWrapper>
+              <ContactPanelProvider>
```

Comment:
Would it be easier to create a component with all these components? so future updates are easier

Reply:
Are you talking about creating a `TestComponent`? I agree, but can we hold off on that to expedite merging this PR?

Reply:
Yes, we can hold off for now so we can push these changes live